vocabularies/gsq-dataset-themes.ttl

```diff
@@ -0,0 +1,100 @@
+@prefix : <http://linked.data.gov.au/def/gsq-dataset-themes/> .

```

Comment:
Still not a fan of using just ```:``` as the prefix. A short prefix makes it easier to read for me.

Reply:
can add in a fix to the template. But as it is a style issue. I'm comfortable leaving as is for this